packages/backend/src/graphql/workHour/mutations/workHourUpdateMutationField.test.ts

```diff
@@ -196,12 +202,13 @@ describe('workHourUpdateMutationField', () => {
     const response = await testServer.executeOperation({
       query: workHourUpdateMutation,
       variables: {
-        id: '1',
         data: {
           date: '2022-01-01',
           duration: 120,
           taskId: '1',
         },
+        date: '2022-01-01',
+        taskId: '1',
       },
     })
 
```

Comment:
I would expect a test here for creating a new work hour, when the work hour does not exist.